portfolio/src/main/webapp/Anime.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <header>
+          <h1>Anime</h1>
+      </header>
+      <nav>
+          <a href="index.html">Home</a>
+          <a href="Anime.html">Anime</a>
+          <a href="family.html">Family</a>   
+      </nav>
+      <div>
+          <h1>List Of My Favourites</h1>
+          <ul>
+              <li>
+                  <h3>Ouran Highschool Host Club</h3>
+                  <p>
+                      Throwback to middleschool weekends with my siblings<br>
+                      This anime was hilarious and great for unwinding and relaxing with the sibs
+                      </p>
```

Comment:
In the cloud shell editor, right click in the main panel and select Format Document to fix these indents.